Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            //Enter information for 2 people
```

Comment:
Instead of doing a comment like this, why don't you just:
```
public static void Demo()
{
  PromptPerson(1);
  PromptPerson(2);
  CloseProgram();
}
```
```
private static void PromptPerson(int index)
{
            Console.WriteLine("Please enter the information for person "+ index);
            NewPerson();
}
```

Reply:
Code should speak for itself 😄

Reply:
Ok yes - understood !  👌